src/screen.cpp

```diff
@@ -300,10 +309,12 @@ void Screen::setVisible(bool visible) {
     if (mVisible != visible) {
         mVisible = visible;
 
-        if (visible)
-            glfwShowWindow(mGLFWWindow);
-        else
-            glfwHideWindow(mGLFWWindow);
+        if (mOwningGLFWContext) {
```

Comment:
If I am managing my own GLFW and windowing code, and just using a pointer to a 'screen' object for my GUI, having the whole window hidden just because I call: screen->setVisible(false) is misleading, what if I just want to hide the GUI alone, but keep my true window visible?